plugins/mysqlx/proto/mysqlx_datatypes.pb.cc

```
@@ -0,0 +1,2360 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: mysqlx_datatypes.proto
+
+#include "mysqlx_datatypes.pb.h"
+
+#include <algorithm>
+
+#include <google/protobuf/io/coded_stream.h>
+#include <google/protobuf/extension_set.h>
+#include <google/protobuf/wire_format_lite.h>
+#include <google/protobuf/descriptor.h>
+#include <google/protobuf/generated_message_reflection.h>
```

Cppcheck (2.20.0)

[error] `#error` This file was generated by a newer version of protoc which is

(preprocessorErrorDirective)